package.json

```diff
@@ -70,16 +70,16 @@
     "@leafygreen-ui/button": "12.0.5",
     "@leafygreen-ui/card": "5.1.3",
     "@leafygreen-ui/checkbox": "6.0.6",
-    "@leafygreen-ui/code": "9.0.0",
+    "@leafygreen-ui/code": "9.1.0",
     "@leafygreen-ui/confirmation-modal": "2.2.2",
     "@leafygreen-ui/expandable-card": "0.9.1",
     "@leafygreen-ui/icon": "11.4.0",
     "@leafygreen-ui/icon-button": "9.1.7",
     "@leafygreen-ui/inline-definition": "2.0.6",
-    "@leafygreen-ui/leafygreen-provider": "2.1.3",
+    "@leafygreen-ui/leafygreen-provider": "^2.2.0",
```

Comment:
Are these updates necessary now that we're not updating Code to v11? [9.1.0](https://github.com/mongodb/leafygreen-ui/blob/main/packages/code/CHANGELOG.md#910) has no dependency updates.